packages/core/src/components/essentials/tooltip.ts

```diff
@@ -57,40 +76,34 @@ export class Tooltip extends Component {
 		// listen to show-tooltip Custom Events to render the tooltip
 		this.services.events.addEventListener(
 			Events.Tooltip.SHOW,
-			(e: CustomEvent) => {
-				const data = e.detail.data || e.detail.items;
-				const defaultHTML = this.getTooltipHTML(e);
-
-				// if there is a provided tooltip HTML function call it
-				if (
-					Tools.getProperty(
-						this.model.getOptions(),
-						"tooltip",
-						"customHTML"
-					)
-				) {
-					tooltipTextContainer.html(
-						this.model
-							.getOptions()
-							.tooltip.customHTML(data, defaultHTML)
-					);
-				} else {
-					// Use default tooltip
-					tooltipTextContainer.html(defaultHTML);
-				}
+			this.handleShowTooltipVar
```

Comment:
why wouldn't this just point to `handleShowTooltip` directly?

Reply:
`handleShowTooltip` can't be access within `addTooltipEventListener()` function. Only `this.handleShowTooltip` can be access.